src/core/integrate.cpp

```diff
@@ -25,6 +25,28 @@
  *  see \ref integrate.hpp "integrate.hpp".
 */
 
+/*
+#########################################
+Fragen:
+-rescale vel/force: warum rescaling?
+-bei calc_forces: warum jedes mal erneut init_forces? 
```

Comment:
init_forces sets the forces to zero (or to whatever comes out of the thermostat if it is turned on). Everything else just adds to them. Otherwise the particles would accumulate forces over time?!